### PR TITLE
Add py3-tensorflow-data-validation

### PR DESCRIPTION
**Pull request by Dentrax**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: py3-tensorflow-data-validation/0002-zetasql-bump-six.patch [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
++Content-Transfer-Encoding: 8bit
++
++Signed-off-by: Furkan Türkal <furkan.turkal@chainguard.dev>
++---
++ bazel/zetasql_deps_step_2.bzl | 6 +++---
++ 1 file changed, 3 insertions(+), 3 deletions(-)
++
++diff --git a/bazel/zetasql_deps_step_2.bzl b/bazel/zetasql_deps_step_2.bzl
++index b222ac3..c1813a2 100644
++--- a/bazel/zetasql_deps_step_2.bzl
```
Comment (jamonation) [inline py3-tensorflow-data-validation/0002-zetasql-bump-six.patch:36]:
```suggestion
+--- bazel/zetasql_deps_step_2.bzl
```

File: py3-tensorflow-data-validation/0002-zetasql-bump-six.patch [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+++++ b/bazel/zetasql_deps_step_2.bzl
```
Comment (jamonation) [inline py3-tensorflow-data-validation/0002-zetasql-bump-six.patch:37]:
```suggestion
++++ bazel/zetasql_deps_step_2.bzl
```

File: py3-tensorflow-data-validation.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+      - py3-installer
+      - py3-gpep517
+      - py3-wheel
+      - bazel-6
+      - bash
+      - openjdk-17
```
Comment (jamonation) [inline py3-tensorflow-data-validation.yaml:37]:
```suggestion
      - openjdk-11
```

File: py3-tensorflow-data-validation.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      - patch
+      - numpy
+      - py3-six
+  environment:
+    JAVA_HOME: /usr/lib/jvm/java-17-openjdk
```
Comment (jamonation) [inline py3-tensorflow-data-validation.yaml:42]:
```suggestion
    JAVA_HOME: /usr/lib/jvm/java-11-openjdk
```

**Review by mamccorm — APPROVED** [review]
great job getting the build passing

**Pull request merged**